packages/typespec-ts/test/unit/parametersGenerator.spec.ts

```diff
@@ -318,11 +317,14 @@ describe("Parameters.ts", () => {
     it("contentTypes has array data defined in form body", async () => {
       const parameters = await emitParameterFromTypeSpec(
         `
+        @encode("binary")
```

Comment:
What if there is no `@encode("binary")` for form-data?

Reply:
Then it would just be a normal request.